README.md

```diff
@@ -45,12 +51,24 @@ SurveyGizmo.configure do |config|
 
   # Optional - Defaults to 50, maximum 500. Setting too high may cause SurveyGizmo to start throwing timeouts.
   config.results_per_page = 100
+
+  # Optional - These configure the Pester gem to retry when SG suffers a timeout or the rate limit is exceeded.
+  # The default number of retries is 0 (AKA don't retry)
+  # retry_interval is in seconds.
+  config.retries = 1
+  config.retry_interval = 60
+  # You can also instruct the gem to retry on ANY exception.  Defaults to false.  Use with caution.
+  config.retry_everything = true
 end
 
-# Retrieve the first page of your surveys
-surveys = SurveyGizmo::API::Survey.all
+# Iterate over your surveys directly with the iterator
+SurveyGizmo::API::Survey.all(all_pages: true).each do |survey|
+  do_something_with(survey)
+end
 # Retrieve ALL your surveys (handle pagination for you)
-surveys = SurveyGizmo::API::Survey.all(all_pages: true)
+surveys = SurveyGizmo::API::Survey.all(all_pages: true).to_a
+# Retrieve the 1st page of your surveys
+surveys = SurveyGizmo::API::Survey.all(page: 1).to_a
```

Comment:
I don't think you should add `.to_a` to `.all` examples, it is best practice to keep the iterators and iterate it after. This way we only query what we need. (if surveys is never used for example, no call will be made)